web_error/error.py

```diff
@@ -23,12 +23,16 @@ def __init__(
         self.message = message
         self.debug_message = debug_message
 
-    def marshal(self: typing.Self) -> dict[str, str]:
-        return {
+    def marshal(self: typing.Self, *, strip_debug: bool = False) -> dict[str, str]:
+        ret = {
             "code": self.code,
             "message": self.message,
             "debug_message": self.debug_message,
         }
+        if strip_debug:
+            ret.pop("debug_message")
```

Comment:
Minor:

```suggestion
            del ret["debug_message"]
```

the key is always present at this point

... or even
```python
if not strip_debug:
    ret["debug_message"] = self.debug_message
```

Reply:
We strip out more in the next PR, otherwise yes :)